src/test/java/io/strimzi/kafka/bridge/utils/Urls.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.utils;
+
+public class Urls {
+
+    private static final String TRANSFER_PROTOCOL = "http://";
+    static final String BRIDGE_HOST = "127.0.0.1";
+    static final int BRIDGE_PORT = 8080;
+
+    private static final String CONSUMERS_PATH = "/consumers/";
+    private static final String INSTANCES_PATH =  "/instances/";
+    private static final String POSITIONS_BEGGINING_PATH =  "/positions/beginning";
+    private static final String POSITIONS_END_PATH =  "/positions/end";
+    private static final String POSITIONS_PATH =  "/positions";
+    private static final String SUBSCRIPTION_PATH =  "/subscription";
+    private static final String TOPICS_PATH = "/topics/";
+    private static final String PARTITIONS_PATH = "/partitions/";
+    private static final String ASSIGMENTS_PATH = "/assignments";
+    private static final String OFFSETS_PATH = "/offsets";
+    private static final String RECORDS_PATH = "/records";
+
+    public static String consumer(String groupId) {
+        return TRANSFER_PROTOCOL + BRIDGE_HOST + ":" + BRIDGE_PORT + CONSUMERS_PATH + groupId;
+    }
+
+    public static String consumerInstances(String groupId, String name) {
```

Comment:
Yes, but I called it because of wanting to have the same names as in the address, for example, address `/consumers/{groupid}/instances/{name}` have names in the plural. SO maybe it's better to have methods name in plural also? @ppatierno WDYT?

Reply:
No it's not in this way. The path represents a resource in the REST API world. You are getting a consumer instance named {name} in the {groupid}. You see "consumers" and "instances" because you are accessing at a group (in consumers) and a specific name instance (in instances). So you are actually getting a single thing, not a collection.